lib/lrug_helpers.rb

```diff
@@ -215,6 +215,41 @@ def has_host?(page)
     content_part_exists?('hosted_by', page) || page.data.has_key?('hosted_by')
   end
 
+  def events_calender(site_url:)
+    calendar = Icalendar::Calendar.new
+     calendar.timezone do |timezone|
+      timezone.tzid = 'Europe/London'
+    end
```

Comment:
[nitpick]

I recognise I am being annoying, but I believe you indented the first line with 1 space too much.